components/ShareScreen.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react';
+import { Share, StyleSheet, SafeAreaView, ScrollView, KeyboardAvoidingView } from 'react-native';
+import styled from 'styled-components/native';
+
+import crafty from '../assets/crafty.png';
+import sunny from '../assets/sunny.png';
+import active from '../assets/active.png';
+import smiley from '../assets/smiley.png';
+
+export const ShareScreen = () => {
+  const [message, setMessage] = useState('');
+  const [mood, setMood] = useState('');
+
+  //Function to implement the Share on phone button, the text content that will be shared is what
+  //appears in the message property on line 20. I am generating a combination of the set Mood, the phrase
+  //explaining that Mood and also the personal message that the user has entered in the TextInput field.
+  const onShare = async () => {
+    try {
+      const result = await Share.share({
+        message: `You've received a ${mood} mood booster! ${phrase()} A special message just for you: ${message}`,
```

Comment:
Idea: A conditional statement that checks if there is a `message` and only print "A special message just for you: " if there is a message